frontend/src/components/Action/ActionNotesList.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Box } from '@material-ui/core'
+
+const notes = [
+    {
+        id: 'hfjkhfkf78f6s9fhj',
+        writerName: 'Anna Therese Asheim',
+        message: 'Kan du se på dette Andreas Rønning?',
+        date: '2021-01-10T11:27:08.024+01:00',
+    },
+]
+
+const ActionNotesList = () => {
+    return (
+        <Box mt={2}>
+            {notes.map(note => {
+                const dateAndTime = note && note.date && new Date(note.date).toLocaleString('en-GB').split(',')
+                const date = dateAndTime[0]
+                const hours = dateAndTime[1].split(':')[0]
+                const minutes = dateAndTime[1].split(':')[1]
```

Comment:
```suggestion
                const noteDate = new Date(note.date)
                
                const date = noteDate.getDate()
                const hour = noteDate.getHours()
                const minutes = noteDate.getMinutes()
```

If we set the types right for the component, we can avoid the `note && note.date &&`.

Reply:
If it's ok with you I will also fix this as part of the  other task